Automattic/wp-calypso
# Plugins: Reduxify plugins browser

#### Changes proposed in this Pull Request

This PR updates the plugins browser to use Redux instead of the legacy `wporg-data` store. It also fixes a discrepancy in the wporg category list reducer, where items could be duplicated when the same request is performed again and again. It cleans up and fixes tests, and last but not least, **removes the last of the plugins flux stores** 🥳 

This PR is part of #24180 and while it mostly resolves it, I'll keep the issue open because there is a bit more polishing to be done to finalize this task.

#### Testing instructions

* Go to `/plugins` and verify all the WP.org plugins lists work the same way as they do in production. Those include:
  * Categories - New, Popular, Featured
  * Search - clicking on Enhancement, Security, Appearance, etc. or a manual search in the search form
  * Pagination on categories (infinite scroll)
  * Anything else you can think of on these.

#### Note

I've removed some WP.com site plugins functionality because we only display plugins for Jetpack sites and not for WP.com simple sites anymore.
